### PR TITLE
Improve bech32 coveralls

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

# Overview

- [ ] I have replaced `isLeft` with `Left ...` here and there to improve coveralls score a bit

# Comments

### CODE REVIEW

File: lib/bech32/test/Codec/Binary/Bech32Spec.hs [hunk 1]
```diff
@@ -273,18 +273,33 @@ validChecksums = map B8.pack
...
-      \sthenumber1andtheexcludedcharactersbio1569pvx"
-    , "pzry9x0s0muk"
-    , "1pzry9x0s0muk"
-    , "x1b4n0q5v"
-    , "li1dgmt3"
-    , "de1lg7wt\xFF"
-    ]
+invalidChecksums :: [(ByteString, Bech32.DecodingError)]
+invalidChecksums =
+            [ ( " 1nwldj5"
```
Comment (KtorZ) [inline lib/bech32/test/Codec/Binary/Bech32Spec.hs]: Sorry to be annoying with this, but let's make this 4-space indent :grimacing: ... Have you configured your editor to pick up the configuration in `.editorconfig` at the root of the project?

**Review by KtorZ — APPROVED** [review]
LGTM, some "indentation" issues though.

File: lib/bech32/test/Codec/Binary/Bech32Spec.hs (hunk 1 above)
Comment (piotr-iohk) [reply lib/bech32/test/Codec/Binary/Bech32Spec.hs]: done. (I must have hit _tab_ too many times...)

**Pull request merged**